src/app/contexts/RequestContext/getStatsDestination/index.js

```diff
@@ -3,10 +3,13 @@ const getStatsDestination = ({ isUK = true, env = 'test', service }) => {
 
   if (service === 'news') {
     destination = isUK !== false ? 'NEWS_PS' : 'NEWS_GNL';
+  } else if (service === 'cymrufyw' || service === 'naidheachdan') {
```

Comment:
you have code climate cognitive complexity issues here, perhaps refactor using a switch instead

Reply:
So something like this?
```
switch (service) {
    case 'news':
      destination = isUK !== false ? 'NEWS_PS' : 'NEWS_GNL';
      break;
    case 'cymrufyw':
    case 'naidheachdan':
      destination = isUK !== false ? 'NEWS_LANGUAGES_PS' : 'NEWS_LANGUAGES_GNL';
      break;
    case 'japanese':
      destination = 'NEWS_LANGUAGES_GNL';
      break;
    default:
      destination = 'WS_NEWS_LANGUAGES';
  }
```

Reply:
yeah, @HarveyPeachey give it a try to see if it improves the complexity

Reply:
It's definitely improved it but it's still one over the limit :(

Reply:
Something like this should have a complexity of 2:
```
const destination = {
  news: isUK ? 'NEWS_PS' : 'NEWS_GNL',
  cymrufyw: isUK ? 'NEWS_LANGUAGES_PS' : 'NEWS_LANGUAGES_GNL',
  naidheachdan: isUK ? 'NEWS_LANGUAGES_PS' : 'NEWS_LANGUAGES_GNL',
  japanese: 'NEWS_LANGUAGES_GNL'
}[service] || 'WS_NEWS_LANGUAGES';

return env === 'live' ? destination : `${destination}_TEST`;
```
Note I haven't tested this code, so do test it if you're going to use it 😊